bulbs/videos/views.py

```diff
@@ -0,0 +1,44 @@
+from django.conf import settings
+from django.http import Http404
+from django.views.generic import DetailView
+from django.views.decorators.cache import cache_control
+
+import requests
+
+
+class SeriesDetailView(DetailView):
+    redirect_correct_path = False
```

Comment:
ah, yeah, this can be removed now. it was a holdover from using `BaseContentDetailView`.